### ISSUE TITLE
CS231n Self-study

### ISSUE THREAD

**Issue by jeongjae96** [jeongjae96/AI#1]
### What?
- [CS231n: Deep Learning for Computer Vision](http://cs231n.stanford.edu/) assignments self-study

**Comment by jeongjae96** [comment]
## Assignment1

### Todo
- [x] k-Nearest Neighbor classifier
- [x] Training a Support Vector Machine
- [x] Implement a Softmax classifier
- [x] Two-Layer Neural Network
- [x] Higher Level Representations: Image Features

**Comment by jeongjae96** [comment]
cross validation이 아닌 grid search를 구현했지만, ```implement cross validation```이라고 잘못 커밋한 것이 많다. 😅

**Comment by jeongjae96** [comment]
## Assignment2

### Todo
- [x] Multi-Layer Fully Connected Neural Networks
- [ ] Batch Normalization
- [ ] Dropout
- [ ] Convolutional Neural Networks
- [ ] PyTorch on CIFAR-10
- [ ] Network Visualization: Saliency Maps, Class Visualization, and Fooling Images
